README.md

```diff
@@ -1,70 +1,75 @@
+  - [Default parameters](#default-parameters)
+  - [What is the ixlib param?](#what-is-the-ixlib-param)
+- [Browser Support](#browser-support)
+- [Meta](#meta)
 
-# imgix.js [![Build Status](https://travis-ci.org/imgix/imgix.js.svg?branch=master)](https://travis-ci.org/imgix/imgix.js)
-
-**This documentation is for imgix.js version `3.0.0` and up. Those using imgix.js `2.x.x` can find documentation in that version's [readme](https://github.com/imgix/imgix.js/tree/2.2.3) and [API reference](https://github.com/imgix/imgix.js/blob/2.2.3/docs/api.md).**
-
-**Note:** If you're looking for a Javascript library just to generate imgix URLs, check out [imgix-core-js](https://github.com/imgix/imgix-core-js).
-
-
-imgix.js allows developers to easily generate responsive images using the `srcset` and `sizes` attributes, or the `picture` element. This lets you write a single image URL that is parsed and used to make images look great at any screen size, by using [imgix](https://imgix.com) to process and resize your images on the fly.
-
-Responsive images in the browser, simplified. Pure JavaScript with zero dependencies. About 2 KB minified and gzipped.
+<a name="overview-and-resources"></a>
+## Overview / Resources
 
-* [Overview / Resources](#overview-and-resources)
-* [Installation](#installation)
-* [Configuration](#configuration)
-* [Usage](#usage)
-  * [`ix-src`](#ix-src)
-  * [`ix-path` and `ix-params`](#ix-path-and-ix-params)
-  * [`picture` tags](#picture-tags)
-* [Advanced Usage](#advanced-usage)
-  * [Overriding `ix-host`](#overriding-ix-host)
-  * [Disabling auto-initialization](#disabling-auto-initialization)
-  * [Manually initializing imgix.js](#manually-initializing-imgix-js)
-  * [`imgix.init()` idempotency](#imgix-init-idempotency)
-  * [Lazy loading With lazysizes](#lazy-loading-with-lazysizes)
-  * [Custom input attributes](#custom-input-attributes)
-  * [Null output attributes](#null-output-attributes)
-  * [Base-64 encoded parameters](#base-64-encoded-parameters)
-  * [Default Parameters](#default-parameters)
-  * [What is the `ixlib` param?](#what-is-the-ixlib-param)
-* [Browser Support](#browser-support)
-* [Meta](#meta)
+**This documentation is for imgix.js version `3.0.0` and up. Those using imgix.js `2.x.x` can find documentation in the corresponding version's [readme](https://github.com/imgix/imgix.js/tree/2.2.3) and [API reference](https://github.com/imgix/imgix.js/blob/2.2.3/docs/api.md).**
 
+**Note:** If a Javascript library for generating imgix URLs is desired, consider using [imgix-core-js](https://github.com/imgix/imgix-core-js) instead.
 
-<a name="overview-and-resources"></a>
-## Overview / Resources
+`imgix.js` allows developers to easily generate responsive images using the `srcset` and `sizes` attributes, or the `picture` element. This lets you write a single image URL that is parsed and used to make images look great at any screen size, by using [imgix](https://imgix.com) to process and resize your images on the fly.
```

Comment:
Can we move this paragraph up so it's the first thing under the "Overview / Resources" header?